.gitattributes

```diff
@@ -1,7 +1,7 @@
 *.sol linguist-language=Solidity
 
 # Automatically collapse generated files in GitHub.
-*.svg linguist-generated
-packages/contract-artifacts/artifacts/*json linguist-generated
-packages/abi-gen-wrappers/wrappers/*.ts liguist-generated
+*.svg linguist-generated=true
+packages/contract-artifacts/artifacts/*json linguist-generated=true
```

Comment:
Hm, if they aren't automatically copied over, the `contract-artifacts` package won't change right? I don't think I follow how this would make us miss something, if nothing changes. Can you elaborate?

Reply:
Sorry, I meant that they aren't automatically copied over right now, as in `contract-artifacts` is updated mostly manually and might be error-prone. So I think if there are any changes, we should be reading over them to make sure nothing unexpected is in them. I would be in favour of ignoring the diffs after we have merged and used https://github.com/0xProject/0x-monorepo/pull/1842 for some time without issues.

Reply:
Ok. This PR isn't actually changing this setting (it was already there). Let's get rid of it from `.gitattributes` in another PR.